app/main.py

```diff
@@ -1 +1,63 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+
+        return 0
```

Comment:
Consider removing the `else` statement by directly returning 0 after the `if` block. This will make the code cleaner and adhere to the checklist item: 'Don't use an extra `else` statement.'